systest/Makefile

```diff
@@ -128,6 +128,10 @@ run_agent_transfers:
 	@echo "running tests extracted from 'f5-openstack-agent' project..."
 	cd scripts && ./disallow_dangerous_vars.sh && ./$@.sh
 
+run_smoke_tests:
+	@echo "running smoke tests for build on PR..."
+	cd scripts && ./disallow_dangerous_vars.sh && ./$@.sh
```

Comment:
disallow dangerous variables goes away in `mitaka` it looks like there won't be a conflict, so this is just an FYI.